docs/ops/state/checkpoints.zh.md

```diff
@@ -26,69 +26,49 @@ under the License.
 * toc
 {:toc}
 
-## Overview
+## 概述
+checkpoint使Flink的状态具有良好的容错性，通过checkpoint机制，Flink可以对作业的状态和计算位置进行恢复，因此Flink作业具备高容错执行语义。
```

Comment:
```suggestion
checkpoint 使 Flink 的状态具有良好的容错性，通过 checkpoint 机制，Flink 可以对作业的状态和计算位置进行恢复。
```

Reply:
thank you for your reviewing, is that I should commit all suggestion here?

Reply:
You can add a new commit for addressing the comments you accept.
If there are some comments not good from your perspective, you can reply it